appendix-basic65.tex

```diff
@@ -486,6 +490,23 @@ \subsection{BACKGROUND}
  13 &    9  &  15   &  9   & light green \\
  14 &    9  &   9   & 15   & light blue\\
  15 &   11  &  11   & 11   & light grey\\
+\hline
+ 16 &   14  &   0   &  0   & guru meditation\\
```

Comment:
we have a colour called that? awesome. I snuck that into the error messages on the last project i worked on

Reply:
Hehe nice, glad it resonates with you too then ;-)